sdk/component/conf.go

```diff
@@ -111,6 +118,15 @@ func (rc *RunnerConfig) isValid() error {
 	return nil
 }
 
+// runnerWithDisabledStoreCheck is an internal only option which is used to disable store checks for components
+// that don't interact with a storage, like targets.
+func runnerWithDisabledStoreCheck(disabled bool) RunnerOption {
```

Comment:
Wouldn't you skip this runner if the storeCheck is `true`? In that case, what do you think about not passing a `(disabled bool)` at all, but have it always false? 

I'm asking this because usually negative function names can be confusing.
I think for clarity it should either be:
```
runnerWithStoreCheck (true/false)
```
or
```
runnerWithDisabledStoreCheck() 
```
(always false, if it's enabled elsewhere by default)